Assets/BossRoom/Scripts/Shared/Net/UnityServices/Lobbies/LocalLobby.cs

```diff
@@ -0,0 +1,321 @@
+using System;
+using System.Collections.Generic;
+using BossRoom.Scripts.Shared.Infrastructure;
+using Unity.Multiplayer.Samples.BossRoom;
+using Unity.Services.Lobbies.Models;
+using UnityEngine;
+
+namespace BossRoom.Scripts.Shared.Net.UnityServices.Lobbies
+{
+    /// <summary>
+    /// A local wrapper around a lobby's remote data, with additional functionality for providing that data to UI elements and tracking local player objects.

```

Comment:
Do I understand this correctly that this is basically a lobby ghost?
"LocalLobby" might be misleading, at first I thought there was two lobbies, one locally managed and one remote. While LocalLobby is just a client mirror of the remote lobby?

Reply:
It is indeed a lobby ghost.